vm-virtio/src/vhost_user/net.rs

```diff
@@ -112,7 +112,7 @@ impl Net {
             kill_evt: None,
             avail_features,
             acked_features,
-            config_space: config_space.clone(),
+            config_space,
```

Comment:
Thanks for fixing this, but please squash it into the commit `Refactor virtio devices handling with VirtioBackend `